webhook/webhook.go

```diff
@@ -0,0 +1,96 @@
+package webhook
+
+import (
+  "crypto/hmac"
+  "crypto/sha256"
+  "encoding/hex"
+  "net/http"
+  "fmt"
+  "errors"
+  "encoding/json"
+  "bytes"
+  "time"
+
+  log "github.com/gophish/gophish/logger"
+)
+
+const (
+  DefaultTimeoutSeconds = 10
```

Comment:
Please add comments for all exported functions and variables in this file. Running `golint` can help you out here.

Reply:
made them private

Reply:
Let's keep them exported. My suggestion was that we need to add comments for them so that the documentation renders them correctly.